README.md

```diff
@@ -3,21 +3,50 @@
 [![Node.js CI](https://github.com/libsql/libsql-client-ts/actions/workflows/ci.yaml/badge.svg)](https://github.com/libsql/libsql-client-ts/actions/workflows/ci.yaml)
 [![License](https://img.shields.io/badge/license-MIT-blue)](https://github.com/libsql/libsql-client-ts/blob/main/LICENSE)
 
-This is the source repository of the JavaScript & TypeScript SDK for libSQL. You can either connect to a local SQLite/libSQL database (embedded in the client) or to a remote libSQL server.
+This is the source repository of the JavaScript & TypeScript SDK for libSQL. You can use it to interact with the following types of databases:
+
+- Local SQLite database files
+- [libSQL sqld] instances (including [Turso])
 
 ## Installation
 
-```
+```shell
 npm install @libsql/client
 ```
 
-## Getting Started
+## Create a database client object
+
+### Importing
 
-Connecting to a local SQLite/libSQL database:
+There are multiple ways to import the module. For Node.js and other environments where want to use a local SQLite [file URL](#local-sqlite-file-urls), as well as network access to `sqld`:
 
 ```typescript
-import { createClient } from "@libsql/client"
+import { createClient } from "@libsql/client";
+```
+
+For environments that don't have a local filesystem, but support HTTP or WebSockets, including:
```

Comment:
Strictly speaking, the `web` import should be used when the environment has *both* HTTP and WebSockets.

Reply:
Is that a hard requirement?  If I only have HTTP available, and I only use an HTTP URL, is there going to be a problem?  Or is the SDK free to use either one as it sees fit?

Reply:
A specific use case here would be the need to share a codebase that gets used in multiple environments, where the developer is dynamically deciding which URL to use. If in some cases they want a websocket and others in http, I'm expecting that this import would be OK.

Reply:
Yes, you are correct. However, if you only plan to use WebSocket, you can import just `@libsql/client/hrana`. Also, if your environment does not define `fetch()`, the `/http` and `/web` import may not work, but `/hrana` should work.